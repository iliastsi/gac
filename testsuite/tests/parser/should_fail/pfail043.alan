hello() : proc
{
  if(true & 3)
    return;
}
