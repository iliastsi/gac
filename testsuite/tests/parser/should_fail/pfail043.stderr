pfail043.alan:3:14: Error: Parse error on input `)'

